### ISSUE TITLE
Update installation instructions.

### ISSUE THREAD

**Issue by HarryAddison** [spacetelescope/astronify#77]
Installation using `pip3 install astronify` fails:

pip3 install astronify
Collecting astronify
  Downloading astronify-0.1.tar.gz (2.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.5/2.5 MB 14.5 MB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
ERROR: Could not find a version that satisfies the requirement thinkx (from astronify) (from versions: none)
ERROR: No matching distribution found for thinkx

It appears that `thinkx` is no longer available through PyPi and should be installed from source (https://github.com/AllenDowney/ThinkX?tab=readme-ov-file).

**Comment by scfleming** [comment]
Thanks I encountered this issue too, and found the same thing. I am not sure Astronify actually needs that package, so if you need to install before a new version of Astronify is released, you can try removing that from the `setup.cfg` and install from source.  As soon as I can I want to try this myself, but am swamped at the moment.

**Comment by folsomcp** [comment]
I just ran into that issue with tinkx too.  It's possible to install directly from their github, e.g.: 
pip install "git+https://github.com/AllenDowney/ThinkX.git"  
But it seems like that github repo hasn't been updated in 7 years, so it would be nicer to not depend on.

**Comment by scfleming** [comment]
Yeah, I have a work-in-progress branch, "py312", that removes this as a dependency.
